### PR TITLE
StopObserver summary typo

### PR DESCRIPTION
**Pull request by riverar**
Minor but bubbles up through IntelliSense and can cause confusion

### CODE REVIEW

File: Assets/HoloToolkit/SpatialMapping/Scripts/SpatialMappingManager.cs [hunk 1]
```diff
@@ -320,4 +320,4 @@ private void UpdateRendering(bool Enable)
             }
         }
     }
-}
```
Comment (keveleigh) [inline Assets/HoloToolkit/SpatialMapping/Scripts/SpatialMappingManager.cs]: Minor formtatting nitpick, but we've tended to leave out a final newline character in this repo.

File: Assets/HoloToolkit/SpatialMapping/Scripts/SpatialMappingManager.cs [hunk 2]
```diff
@@ -320,4 +320,4 @@ private void UpdateRendering(bool Enable)
...
\ No newline at end of file
+}
```
Comment (keveleigh) [inline Assets/HoloToolkit/SpatialMapping/Scripts/SpatialMappingManager.cs]: Minor formtatting nitpick, but we've tended to leave out a final newline character in this repo.

Comment (riverar) [inline Assets/HoloToolkit/SpatialMapping/Scripts/SpatialMappingManager.cs]: Was GitHub automagic, will manually edit and rebase.

**Comment by riverar** [comment]
Removed errant EOF newline.

**Pull request merged**